backend/core/src/listings/listings.controller.ts

```diff
@@ -133,4 +133,18 @@ export class ListingsController {
     await this.listingsService.delete(listingId)
     await clearCacheKeys(this.cacheManager, [...this.cacheKeys, `/listings/${listingId}`])
   }
+
+  @Put(`:listingId/close`)
+  @ApiOperation({ summary: "Close listing by id", operationId: "close" })
+  async close(@Param("listingId") listingId: string) {
+    await this.listingsService.close(listingId)
+    await clearCacheKeys(this.cacheManager, [...this.cacheKeys, `/listings/${listingId}`])
+  }
+
+  @Put(`:listingId/unpublish`)
+  @ApiOperation({ summary: "Unpublish listing by id", operationId: "unpublish" })
+  async unpublish(@Param("listingId") listingId: string) {
+    await this.listingsService.unpublish(listingId)
+    await clearCacheKeys(this.cacheManager, [...this.cacheKeys, `/listings/${listingId}`])
+  }
```

Comment:
@dominikx96 and @pbn4 , do we really want to maintain separate endpoints for closing, unpublishing, etc? If it does make sense to have endpoints to change the status, can we have one with a status param, like `@Put(':listingId/:status')`?